test/TestDataFactory.js

```diff
@@ -0,0 +1,38 @@
+// ## TestDataFactory functions
+// Emulate old, pre-RDF/JS output (plain strings instead of Terms) to make tests pass
+
+// ### Creates an IRI
+function namedNode(iri) {
+  return iri;
+}
+
+// ### Creates a blank node
+function blankNode(name) {
+  return '_:b' + name;
+}
+
+// ### Creates a literal
+function literal(value, tag) {
+  if (tag) {
+    if (tag.length < 10) // hacky "heuristic" for test purposes only
```

Comment:
This will break down with short datatype URIs of course :) But good enough to make the tests pass.